mobile-app/src/components/PfpNameUsernameBoxes.tsx

```diff
@@ -71,14 +77,17 @@ export default function ProfileCreationBoxes({
           }}
           ref={refUsernameInput}
           value={username}
-          onChangeText={(text) => setUsername(text)}
+          onChangeText={(text) => {
+            setUsername(text);
+            setFormData({ ...formData, username: text });
+          }}
           onSubmitEditing={() => {
             refEmailInput.current?.focus();
           }}
         ></TextInput>
       </View>
       {/* Email */}
-      {editPage == true && (
+      {/* {editPage == true && (
```

Comment:
why?

Reply:
We dont store email anywhere in the DB (except for an exception with firebase auth thru email signin) but its not consistent. Keeping it commented for now but its easy to add email requirement in the future